src/__tests__/components/fields/text-field/text-field.spec.tsx

```diff
@@ -0,0 +1,240 @@
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+import { FrontendEngine } from "../../../../components";
+import { IEmailFieldSchema, INumericFieldSchema, ITextFieldSchema } from "../../../../components/fields";
+import { IFrontendEngineData } from "../../../../components/types";
+import {
+	ERROR_MESSAGE,
+	FRONTEND_ENGINE_ID,
+	TOverrideSchema,
+	getErrorMessage,
+	getField,
+	getSubmitButton,
+	getSubmitButtonProps,
+} from "../../../common";
+
+const submitFn = jest.fn();
+const componentId = "field";
+const componentLabel = "Textfield";
+const defaultFieldType = "text-field";
```

Comment:
suggest you use uppercase for static consts

Reply:
ok have updated for all test suites